### DISCUSSION TITLE
支持以免费登录方式启动我的世界基岩版

### DISCUSSION THREAD

**Discussion by MJIFmc** [Meloong-Git/PCL#379] [提问] [answered]
一般可以通过修改系统dll文件或修改内存方式做到，但网上能找到的工具仍然较少
对于没有购买基岩版的玩家，这也许很重要

**Comment by idhaname** [comment, marked as answer]
PCL2 不会添加对基岩版的任何支持。
请看 #81

**Comment by WForst-Breeze** [comment]
考虑到还时不时有人来问我这种鬼事情，那我再吼一句……

### 这玩意儿我根本启动不了！
### 用官方启动器吧！
### 告辞！
——龙猫（[#81](https://github.com/Hex-Dragon/PCL2/issues/81)）

**Comment by MJIFmc** [comment]
我已经买了
















在 2022-11-26 09:44:49，"lvjiajjjlzy" ***@***.***> 写道：

没钱玩你妈逼，死

**Comment by MJIFmc** [comment]
...



在 2022-11-13 18:28:31，"WForstBreeze" ***@***.***> 写道：

考虑到还时不时有人来问我这种鬼事情，那我再吼一句……

这玩意儿我根本启动不了！
用官方启动器吧！
告辞！

——龙猫（#81）
